mmv1/templates/terraform/examples/secret_with_automatic_cmek.tf.erb

```diff
@@ -0,0 +1,21 @@
+data "google_project" "project" {}
+
+resource "google_kms_crypto_key_iam_member" "kms-secret-binding" {
+  crypto_key_id = "<%= ctx[:vars]['kms_key_name'] %>"
+  role          = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
+  member        = "serviceAccount:service-${data.google_project.project.number}@gcp-sa-secretmanager.iam.gserviceaccount.com"
+}
+
+resource "google_secret_manager_secret" "<%= ctx[:primary_resource_id] %>" {
+  secret_id = "<%= ctx[:vars]['secret_id'] %>"
+
+  replication {
+    automatic {
+      customer_managed_encryption {
+        kms_key_name = "<%= ctx[:vars]['kms_key_name'] %>"
```

Comment:
Where is the kms_key and keyring defined?
For automatic replication location is global and CMEK should also be defined in a global keyring.

Reply:
I've used the function `BootstrapKMSKey` as mentioned in the [Secret.yaml](https://github.com/GoogleCloudPlatform/magic-modules/pull/8662/files#diff-9001ca5000d45e69e6d14ce2f520e2e82e8a77c74fde2affd94bafdc8dd29db3R50) that will use a particular keyring and a particular key i.e. `tftest-shared-keyring-1` and `tftest-shared-key-1` respectively in the test environment. The implementation of this function can be found here: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/acctest/bootstrap_test_utils.go#L42.